HeliPort.xcodeproj/project.pbxproj

```diff
@@ -65,16 +68,16 @@
 
 /* Begin PBXFileReference section */
 		2B879D3A8D737E5EF06C350F /* Pods-HeliPort.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-HeliPort.release.xcconfig"; path = "Target Support Files/Pods-HeliPort/Pods-HeliPort.release.xcconfig"; sourceTree = "<group>"; };
+		42979EFF24B3A27200F15FBD /* el */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = el; path = el.lproj/Localizable.strings; sourceTree = "<group>"; };
```

Comment:
Before I merge it in, can you remove the unnecessary change here? It might cause conflict with other PRs.

Reply:
The reference id is the same so no conflict. Moving lines around in `pbxproj` is just the crap Xcode does sometimes. I have a suspicion that this file was edited manually and Xcode just tries to "correct" it.

Reply:
OK. Thanks for the great work.